README.md

```diff
@@ -79,26 +80,27 @@ See the new [growlab app](/app) for your Raspberry Pi
 
 | Name  | Bronze   | Silver   | Gold   | Platinum |
```

Comment:
What happened to this section?

Reply:
removed some spaces and did some alignment in the code, I can revert

Reply:
It would be easier to merge if you just added your line(s) yep thanks

Reply:
done